stripe.go

```diff
@@ -250,6 +263,7 @@ func (s *BackendConfiguration) CallMultipart(method, path, key, boundary string,
 // NewRequest is used by Call to generate an http.Request. It handles encoding
 // parameters and attaching the appropriate headers.
 func (s *BackendConfiguration) NewRequest(method, path, key, contentType string, body io.Reader, params *Params) (*http.Request, error) {
+	method = strings.ToUpper(method)
```

Comment:
Did you end up putting in the call to `ToUpper` for any particular reason? I just grepped the library and it seems like all verbs are specified as uppercase already, so I don't think we need this.

It also occurred to me that we should really just be using the `http` constants like `http.MethodPut` instead of directly passing verb names. I'll put together a patch that does that separately.

Reply:
And just to be clear, I'd have a preference for dropping `ToUpper` for now if it's not needed.

Reply:
I'm moving us to `net/http` constants in #587.